content/terms-of-service/termsOfService.md

```diff
@@ -1,3 +1,494 @@
 ---
-title: "Terms Of Service"
----
\ No newline at end of file
+title: "Tardigrade Terms Of Service"
+lastUpdated: "March 19, 2020"
+---
+
+<p>THESE TARDIGRADE TERMS OF SERVICE (THE “<b>AGREEMENT</b>”) ARE A LEGAL AGREEMENT BETWEEN YOU OR THE ENTITY YOU REPRESENT (“<b>USER</b>,” OR “<b>SUBSCRIBER</b>,” OR “<b>YOU</b>” OR “<b>YOUR</b>”) AND STORJ LABS INTERNATIONAL, SEZC,, INCLUDING ITS PARENT, SUBSIDIARIES, AFFILIATES, DIRECTORS, OFFICERS, EMPLOYEES, AGENTS, LICENSEES, AND EMPLOYEES (“<b>COMPANY</b>,” AS DEFINED BELOW, “<b>US</b>,” “<b>WE</b>” OR “<b>OUR</b>”) GOVERNING THE TERMS OF YOUR  USE OF THE STORAGE SERVICES AS DEFINED HEREIN.  THIS AGREEMENT HEREBY INCORPORATES BY REFERENCE THE FOLLOWING AGREEMENTS: <a target="_blank" href="https://storj.io/privacy-policy/" class="link">PRIVACY POLICY</a>, <a target="_blank" href="https://storj.io/terms-of-use/" class="link">TERMS OF USE</a>, <a target="_blank" href="https://storj.io/code-of-conduct/" class="link">CODE OF CONDUCT</a>, AND <a target="_blank" href="https://storj.io/storj-share-terms/" class="link">STORAGE NODE OPERATOR TERMS AND CONDITIONS.</a>  BY ACCEPTING THE TERMS OF THIS AGREEMENT, EITHER BY CLICKING TO SIGNIFY ACCEPTANCE, BY SETTING UP A USER ACCOUNT, OR BY USING THE STORAGE SERVICES, YOU AGREE TO BE BOUND BY THE PROVISIONS OF THIS AGREEMENT.   YOU REPRESENT AND WARRANT THAT YOU HAVE THE AUTHORITY TO ENTER INTO THIS AGREEMENT ON USER’S BEHALF. </p>
+<br>
+<p><b>PLEASE READ THE TERMS OF THIS AGREEMENT CAREFULLY TO ENSURE THAT YOU UNDERSTAND EACH PROVISION. THIS AGREEMENT CONTAINS A BINDING ARBITRATION AND WAIVER OF CLASS ACTION RIGHTS PROVISION THAT REQUIRES THE USE OF INDIVIDUAL ARBITRATION TO RESOLVE DISPUTES AS SET FORTH BELOW.  IF YOU DO NOT AGREE TO THE TERMS OF THIS AGREEMENT, PLEASE DO NOT ACCESS OR USE THE STORAGE SERVICES.</b></p>
+<br>
+<p><b>IMPORTANT: THERE IS NO GUARANTEE THAT ANY MATERIALS STORED VIA THE STORAGE  SERVICES WILL BE AVAILABLE ON DEMAND OR RETRIEVABLE AT ANY TIME.  YOU ARE RESPONSIBLE FOR MAINTAINING BACKUP COPIES OF ALL STORAGE MATERIALS AT ALL TIMES. STORJ IS NOT RESPONSIBLE FOR ANY LOSS OR DAMAGE TO STORAGE MATERIALS.</b></p>
+<br>
+<p>1.&nbsp;&nbsp;<u>Definitions</u>.  For purposes of this Policy, the following terms have the meanings set forth below.</p>
```

Comment:
"1. Definitions" - bold not underline

Reply:
<img width="764" alt="Screen Shot 2020-03-27 at 4 34 47 PM" src="https://user-images.githubusercontent.com/10093024/77798358-15ddc380-7049-11ea-813c-7c05e45095b5.png">